recipe/meta.yaml

```diff
@@ -254,30 +308,29 @@ outputs:
         - python >=3.10
         - pip
         - cython <3
-        - custatevec  ={{ cusv_version }}
-        - cutensornet ={{ cutn_version }}
+        - custatevec  =1.7.0
+        - cutensornet =2.6.0
+        - cudensitymat =0.0.5
         - packaging
       run:
         - python >=3.10
-        - numpy  >=1.21,<2
+        - numpy  >=1.21
         - cupy   >=13.0
         - {{ pin_subpackage('custatevec', max_pin='x') }}
         - {{ pin_subpackage('cutensornet', max_pin='x') }}
-        - cuda-version >=11.0.*,<13.a0  # again, cuQuantum Python is "CUDA agnostic"
+        - {{ pin_subpackage('cudensitymat', max_pin='x') }}
+        - cuda-version >={{ cuda_major }}.0,<{{ cuda_major + 1 }}
```

Comment:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1109631&view=logs&j=d93976e3-6ed1-588b-ebf8-e8a19c2becc4&t=8f815395-247d-5523-e57f-dfa3d88394bc&l=2840

Reply:
The python package should be linked dynamically when pip installing it. Since the build is successful, I am not sure what's happening. Does `$PREFIX/lib` not point to the same directory as the exported libraries?

Reply:
I suggest checking the *.cpython-312-x86_64-linux-gnu.so files using ldd to see if they declare dynamic links on the cuquantum C-API libraries. If not, you're probably missing linker arguments.

Maybe these [extension module declarations](https://github.com/NVIDIA/cuQuantum/blob/b3e6d623ecb7d95621660d64f864e09c93c22028/python/setup.py#L79) are missing the [`libraries` key?](https://setuptools.pypa.io/en/latest/userguide/ext_modules.html#extension-api-reference)

Reply:
They don't declare dynamic links. I checked the previous conda release package, as well as pypi and local builds for the current release and none of them declare dynamic links.

Reply:
Thanks @leofang for the clarification! Resolving since the cuquantum-python output will be removed.